CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 This contains only the most important and/or user-facing changes; for a full changelog, see the commit history.
 
+## [2.5.0](https://github.com/ably/ably-js/tree/2.5.0) (2024-10-31)
+
+With this release, ably-js will now expose the new `Message` attributes needed to support upcoming features in the Ably service.
```

Comment:
do we want to mention what kind of "features"? like message edits/deletes/annotations